web/sidebars.js

```diff
@@ -14,8 +14,8 @@ module.exports = {
     {
       type: 'category',
       label: 'Tutorial',
-      collapsed: false,
-      collapsible: false,
+      collapsed: true,
```

Comment:
We kinda lost the _overview of all the topics_ with everything being collapsed. We had this in the old docs if I'm not mistaken and we decided to keep them expanded in this latest iteration.

For example, data model is not indicative of operations or advanced is not indicative of jobs. You have seen _jobs_ and now you want to know more, but you have to expand multiple sections to find them.

![Screenshot 2024-05-28 at 11 01 53](https://github.com/wasp-lang/wasp/assets/2223680/2baacfc5-941b-48ea-9dc9-03599e7f6e7f)

Reply:
Yeah, but there was so much stuff you really couldn't see anything. So instead of having a top level overview, you had a glimpse into 20% of the docs and that is it.

This way you can scan and find your way around.

If we have important stuff we want to take up, we can do that by organizing a new category. For example, take Operations to the top level. Or, we can put data model and ops and more stuff under "Foundations", and then the rest under "Advanced".

So I am not saying we can't improve this, but I think the old solution stopped working at some point when the docs grew, and this is the "future".

Reply:
> Yeah, but there was so much stuff you really couldn't see anything. So instead of having a top level overview, you had a glimpse into 20% of the docs and that is it.

It was more like 50%, but that's fine since users know how to scroll. It's much easier to scroll than to open 5 sub menus, not to mention the feelings you get while looking for something you can't find or even worse not knowing where to start.

I'll let @sodic comment of this as well, but we kinda took this from Vue.js docs: https://vuejs.org/guide/introduction.html since they were the gold standard in our eyes.

Reply:
We could in theory go somewhere in between: have some categories collapsed by default (like Tutorial and Auth, those don't benefit from being open), and some open by default (e.g. Advanced).
Although it seems to me like some kind of reorganization would be the best solution.

Reply:
@infomiho for me it was a huge mess on the left, very hard to get any kind of an overview, or what is important or what is not. I keeping having that issue very time I look at the docs. This finally enables me to navigate the top level and dive to the depth I need.

Reply:
I dropped this part our of this PR to be discussed separately!